app/admin/tenants/tenants.py

```diff
@@ -123,13 +69,13 @@ async def create_tenant(
             key_management_mode="managed",
             label=body.name,
             wallet_key=token_urlsafe(48),
-            wallet_name=get_wallet_name_for_role(tenant_role),
+            wallet_name=uuid4().hex,
```

Comment:
Is this really what we want here? not a human readable name?

Reply:
The wallet name is something internal and more of an id to the wallet. If you look at the indy-sdk docs you can see it uses id instead of name: https://github.com/hyperledger/indy-sdk/tree/master/wrappers/nodejs#createwallet--config-credentials----void. Not sure why ACA-Py chose to use name here. A name suggests to me there can be multiple wallets with the same name, but that's not the case, so it acts more of an id here

Reply:
ah alright thx for the pointer i'll read up on that